### ISSUE TITLE
Latest stable release alias

### ISSUE THREAD

**Issue by acobster** [lando/lando#810]
Feature Request
===============

As a lando user who runs automated builds per the [Lando and CI](https://docs.devwithlando.io/tutorials/lando-and-ci.html) docs, I want to hard-code a download URL for the latest *stable* release of Lando in my `.travis.yml`.

Please provide a `latest-stable` download URL that can be safely used for scripting:

```
before_install:
  - sudo apt-get -y update
  - sudo apt-get -y install cgroup-bin curl
  - curl -fsSL -o /tmp/lando-latest.deb http://installer.kalabox.io/lando-latest-stable.deb
  - ...
```

Thanks!

**Comment by devtype** [comment]
I would like this too it makes upgrading simpler.

**Comment by pirog** [comment]
I think the easiest way for us to do this is to upload a `lando-latest-stable.PKGTYPE` to s3 and use that vs aliasing a stable url to some github release. 

On that theme, we also should be pushing up test builds to s3 on non-master branches just to verify the pipe to s3 is intact

**Comment by macjules** [comment]
@pirog Any update on this please? I need to be able to reference the latest build using a JAMF applications installer. I have a few developers waiting on this as a matter of urgency.

**Comment by pirog** [comment]
@macjules we've been focused on getting RC1 out the door so afaik there is no update on this.

**Comment by macjules** [comment]
@pirog  Understood. As a temporary measure we will clone to an S3 bucket and make the dmg file available from there.

**Comment by acobster** [comment]
w000000t! Thank you, thank you!

**Comment by sirtet** [comment]
> This is not going to be documented anywhere except here
Hmm, why that? I try installing Lando for the first time, and  the Docs here
https://docs.devwithlando.io/installation/linux.html
talk about a `lando-stable.deb`, but the link to the releases page brings up only alpha- beta- rc- 3.0.0.0 at the moment... 
Am i stoopid or missing something?

**Comment by gitressa** [comment]
I was looking at [Hyperdrive](https://github.com/lando/hyperdrive), and in the file https://github.com/lando/hyperdrive/blob/master/installers/debian.sh I noticed there was an alias link for the latest stable release:

```
  # Install lando if needed
  if [[ $LANDO_INSTALLED == "false" ]]; then
    clean_apt
    echo -e "Downloading Lando"
    curl -f#SL -o /tmp/lando.deb https://files.devwithlando.io/lando-stable.deb
```

If this alias is by now a permanent fixture, perhaps this could be added under step 2 in the [documentation](https://docs.lando.dev/basics/installation.html#linux)?

> 2. Download the latest .deb, .pacman or .rpm package from GitHub
> `wget https://files.devwithlando.io/lando-stable.deb`
> 3. Run the required package installation command for you ...

I would prefer that, over having to find the latest version myself. This would also make it easier for new users.

**Comment by pirog** [comment]
I dont think this is something we want to do. This is partly so we can track and get metrics on things but also because `https://files.devwithlando.io` is something we pay for bandwidth wise.

It's also not clear to me that asking the user, even a linux user, to run a bunch of CLI commands is "easier" than downloading a deb and double clicking it, which im guessing is what most people do regardless of what we put in the instructions here. 

That said, its definitely common for linux things to have multiple installation pathways, it would be cool if we could say "download and double click" or "run these commands" or, ideally, "install from this ppa"

**Comment by pirog** [comment]
this is all to say that i am on board with the spirit of your proposal to make things easier ;)

**Comment by gitressa** [comment]
Great, I am glad that we agree that it should be as easy as possible :-)

> https://files.devwithlando.io is something we pay for bandwidth wise

I see. That is a factor to weigh in ... And I agree that using a [ppa](https://help.launchpad.net/Packaging/PPA) would be an ideal solution, since it would allow Lando to simply get updated together with the rest of the system, like for example Docker already does.

I don't know how many people click a deb-file and install it like that ... I always `wget` first and then `sudo dpkg -i ...`

Would it be possible to add it under https://docs.lando.dev/basics/installation.html#debian, where `lando-stable.deb` is already mentioned?
```
wget https://files.devwithlando.io/lando-stable.deb
sudo dpkg -i lando-stable.deb
```
PS. Entirely unrelated, but stale bot closed this issue a while ago, which I wouldn't want to create a new issue about: #1847. I am not sure how to proceed ... Is it possible to reopen it, since @generalredneck is also looking for this feature?

**Comment by pirog** [comment]
@gitressa you are free to open up another issue to improve the documentation since that is a separate issue to this one. 

Stalebot did not close this issue, i did, and i closed it because the issue was satisfied by 
https://github.com/lando/lando/issues/810#issuecomment-459839067
as evidenced by both the links and the 3 units of love that were given

**Comment by gitressa** [comment]
Sorry, I wasn't precise enough. I meant this issue: #1847 was closed. The current issue (#810) on this page I understand was closed, and why.

**Comment by pirog** [comment]
Gotcha. My bad, not yours, i did not read your comment carefully enough! clear ref to #1847, probably just assumed that was the issue number for this thread ;)

**Comment by gitressa** [comment]
No problem, you have a lot of issues, releases, etc. for you to keep track of, and you are doing a tremendous job. I forgot to mention it, but big thanks for maintaining Lando, it's indispensable. And the latest release is so much faster in all aspects, like starting up and shutting down. So thanks! By the way, what does `aft` stand for?

**Comment by gitressa** [comment]
About the confusion: I really like the way drupal.org turns a referenced issue into a link with the actual title of the issue (like `[#12345678]` > "The issue title, linked"), GitHub really should consider doing this as well.

I have created a PR which adds the download link alias for Debian stable: #2078.
